packages/autocomplete-core/src/createAutocomplete.ts

```diff
@@ -53,6 +56,33 @@ export function createAutocomplete<
       navigator: props.navigator,
       ...setters,
     });
+
+    if (
+      !isAlgoliaInsightsPluginEnabled &&
```

Comment:
Even if the Insights plugin is added L101 (due to `insights: true`), wouldn't this still be `false` because it has already been computed?

Reply:
Good call, I thought about it and somehow forgot. I changed it and added a test [6d2cb69](https://github.com/algolia/autocomplete/pull/1205/commits/6d2cb69395ec8fdd7e7d09b9d7fbfc52d06fb116).